3rdparty/filament/filament_build.cmake

```diff
@@ -39,15 +41,21 @@ if(APPLE)
     endif()
 endif()
 
+if(LINUX_AARCH64)
+    set(lib_dir lib/aarch64)
+endif()
+
 set(lib_byproducts ${filament_LIBRARIES})
 list(TRANSFORM lib_byproducts PREPEND ${FILAMENT_ROOT}/${lib_dir}/${CMAKE_STATIC_LIBRARY_PREFIX})
 list(TRANSFORM lib_byproducts APPEND ${CMAKE_STATIC_LIBRARY_SUFFIX})
 
 ExternalProject_Add(
     ext_filament
     PREFIX filament
-    URL https://github.com/isl-org/filament/archive/d1d873d27f43ba0cee1674a555cc0f18daac3008.tar.gz
-    URL_HASH SHA256=00c3f41af0fcfb2df904e1f77934f2678d943ddac5eb889788a5e22590e497bd
+    GIT_REPOSITORY https://github.com/isl-org/filament
```

Comment:
- Pin the commit id
- Download `tar.gz` instead of using git clone, so that the download can be cached.

Reply:
Done.